benchexec/tools/korn.py

```diff
@@ -34,8 +37,21 @@ def version(self, executable):
     def name(self):
         return "Korn"
 
-    def project_url(self):
-        return "https://github.com/gernst/korn"
+    def cmdline(self, executable, options, task, rlimits):
+        cmd = [executable]
+        cmd = cmd + options
+
+        data_model = task.options.get("data_model")
+
+        if data_model == "ILP32":
+            cmd = cmd + ["-32"]
+
+        if data_model == "LP64":
+            cmd = cmd + ["-64"]
+
+        cmd = cmd + [task.single_input_file]
```

Comment:
Compared to the previous implementation, this restricts the tool-info module to tasks with exactly one input file. But I guess that the tool does not support other tasks anyway?

Reply:
Yes, that is intended